src/uct/api/uct.h

```diff
@@ -1009,6 +1009,68 @@ ucs_status_t uct_wakeup_signal(uct_wakeup_h wakeup);
 
 /**
  * @ingroup UCT_RESOURCE
+ * @brief Disable synchronous progress for the interface
+ *
+ * Notify the transport that it should avoid doing anything
+ * during @ref uct_worker_progress(). Thus the latency of
+ * other transports may be reduced.
+ *
+ * To progress the interface the application must call
+ * @ref uct_iface_async_progress(). 
+ * 
+ * The function can be called from any context or thread.
+ *
+ * By default the progress is enabled when interface is created.
```

Comment:
minor re-wording suggested for clarity:
By default, progress is enabled when the interface is created.